consensus/interface.go

```diff
@@ -153,6 +153,7 @@ type ScheduledProcessor interface {
 type SignatureHandler interface {
 	Reset(pubKeys []string) error
 	CreateSignatureShare(msg []byte, index uint16, epoch uint32) ([]byte, error)
+	CreateSignatureShareWithPrivateKey(message []byte, index uint16, epoch uint32, privateKeyBytes []byte) ([]byte, error)
```

Comment:
this needs to be refactored to remove private key as parameter.
If you add a TODO I can do the refactor.

Reply:
added TODO